src/tests/Login.test.jsx

```diff
@@ -107,3 +107,23 @@ it('handles wrong credentials', async () => {
     
     expect(screen.getByLabelText('errorMessage')).toHaveTextContent("The server is currently unavailable. Try again later")
   })
+
+it('input field', async () => {
+    render(<Login />)
+
+    fireEvent.change(screen.getByPlaceholderText('Username or Email'), {
+        target: { value: 'MyUsername' },
```

Comment:
If this is for checking empty field, the target value should be "")

Reply:
i tried to do that but then the tests are not passing and showing me the error

Reply:
@khushishikhu, perhaps you should use the [.toBeRequired()](https://github.com/testing-library/jest-dom#toberequired) check of @tesing-library/jest-dom instead?